app/src/main/java/com/codesoom/assignment/TaskHttpHandler.java

```diff
@@ -0,0 +1,158 @@
+package com.codesoom.assignment;
+
+import com.codesoom.assignment.models.Task;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.*;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class TaskHttpHandler implements HttpHandler {
+    ObjectMapper objectMapper = new ObjectMapper();
+    private List<Task> tasks = new ArrayList<>();
+
+    @Override
+    public void handle(HttpExchange exchange) throws IOException {
+        String content = "OK";
+
+        String method = exchange.getRequestMethod();
+        URI uri = exchange.getRequestURI();
+        String path = uri.getPath();
+
+        System.out.println(method + " " + path);
+
+        // "/tasks/1" is split into { "", "tasks", "1" }
+        String[] segments = path.split("/");
+
+        if (segments.length < 2 || segments.length > 3 || !segments[0].isBlank() || !segments[1].equals("tasks")) {
+            // 404 NotFound Exception
+            System.out.println("[Invalid URI] 404 NotFound Exception");
+            exchange.sendResponseHeaders(404, 0);
+            OutputStream outputStream = exchange.getResponseBody();
+            outputStream.flush();
+            outputStream.close();
+            return;
+        }
+
+        int id = 0;
+        if (segments.length == 3) {
+            // if segments[2] is not integer, 400 BadRequest - NumberFormatException thrown
+            id = Integer.parseInt(segments[2]);
+        }
+
+        InputStream inputStream = exchange.getRequestBody();
+        String body = new BufferedReader(new InputStreamReader(inputStream))
+                .lines()
+                .collect(Collectors.joining("\n"));
+
+        if (method.equals("GET") ) {
+            if (segments.length == 2) {
+                content = taskToJSON();
+            }
+
+            if (segments.length == 3) {
+                boolean isthere = false;
+                for (Task task : tasks) {
+                    if(task.getId() == id) {
+                        isthere = true;
+                        content = taskToJSON(id);
+                        break;
+                    }
+                }
+
+                if (isthere == false) {
```

Comment:
다음과 같이 쓰는 쪽이 더 단순합니다.
```suggestion
                if (!isthere) {
```